index.html

```diff
@@ -155,6 +157,80 @@ <h2>
         </p>
       </div>
     </section>
+    <section> <h3>Permissions and user prompts</h3>
+      <p>
+        The term <a>obtain permission</a>
+        is used for acquiring trust for a given operation.
+      </p>
+      <p>
+        The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
+        via user interface or setting or host device platform features, using which
+        the user approves the permission of a <a>browsing context</a> to access the
+        given functionality.
+      </p>
+      <p>
+        All <a>expressed permission</a>s that are preserved beyond the current
+        browsing session MUST be revocable.
+      </p>
+      <p>
+        The term <dfn id="askforgiveness">ask for forgiveness</dfn> refers to some
+        form of unobtrusive notification that informs the user of an operation
+        while it is running.
+        UAs SHOULD provide the user with means to ignore similar future
+        operations from the same <a>origin</a> and advertise this to the user.
```

Comment:
By "ignore similar future operations" do you mean that the user is given the option to hide the indication that the operation is running or that the user is given the option to deny the origin the ability to perform the operation in the future. The spec should probably mention both.

Reply:
Let me spec both